app/views/content/life-as-a-teacher/explore-subjects/biology/_article.html.erb

```diff
@@ -96,14 +96,15 @@
 
   <section class="col col-space-m col-720">
     <h3>Fund your teacher training</h3>
+    <%= render Content::InsetTextComponent.new(text: "The figures on this page apply if you're starting your training $bursaries_fundingdates$. Financial support for courses starting the following year will be announced in autumn 2024.") %>
     <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_biology %> are available for eligible trainee biology teachers</p>
```

Comment:
Nothing to do with holding message but there is no closing `</a>` and it is missing a full-stop outside the link text - this needs updating on more than one subject page so suggest we do this retrospectively as we do other subject page changes

cc @Sarah-DfE to add to the subject page ticket!